scarb/src/ops/metadata.rs

```diff
@@ -135,6 +137,11 @@ fn collect_package_metadata(package: &Package) -> m::PackageMetadata {
 
     let edition = edition_variant(package.manifest.edition);
 
+    let experiments = package.manifest.experiments.clone().unwrap_or_default();
+    let experimental_features = json!(ExperimentalFeaturesConfig {
+        negative_impls: experiments.contains("negative_impls"),
+    });
+
```

Comment:
no need to do that here, we should just pass-through what's contained within Scarb.toml